src/theme/utils.ts

```diff
@@ -46,12 +46,13 @@ export const iterateObject = <T>(
   func: (value: string, name: string) => generatedColorShades | string
 ): Record<string, unknown> =>
   Object.keys(obj).reduce((acc, value) => {
-    acc[value] =
-      typeof obj[value] !== 'object'
-        ? func(obj[value], value)
-        : EXCLUDED.includes(value)
-        ? obj[value]
-        : iterateObject<TextPaletteConfigType | flatPaletteConfigType>(obj[value], func);
+    if (typeof obj[value] !== 'object') {
```

Comment:
Possibly out of scope, but maybe this util method would benefit from having a comment explain what its purpose is?